doc/configuration.rst

```diff
@@ -447,7 +447,7 @@ traceback_max_length
 Parameters controlling the contents of batch notifications sent from the
 scheduler
 
-email_interval
+email_interval_minutes
```

Comment:
this config variable should remain `email_interval`, that's the name which is searched for by the code https://github.com/spotify/luigi/blob/857deff3c7ade4bfd57f9acfb0e4816814cce8c5/luigi/batch_notifier.py#L17

Reply:
I made a change in that file. But looks like `tox` through it out. I didn't know I have to commit before running `tox`. Please check the commit I'll make in a few minutes.

Reply:
Committed. I'm not sure if it solves the issue though. And I don't know how I'd go about writing a test for it. Do you have any tips?
https://github.com/spotify/luigi/pull/3125/files#diff-b5f8cc1a0854bdb1861ce3c50dea58a7b7759c4a36ca6a65a1516fa47d39e7ffL18-R20

Reply:
Cool; yeah, this is what i was expecting.

As for the link/code preview, i think it has to be a code link from `master`. I didn't do anything special other than past the link.